pkg/aws/ec2/mock/mock_test.go

```diff
@@ -88,76 +79,76 @@ func (e *MockSuite) TestMock(c *check.C) {
 	api.UpdateSecurityGroups([]*types.SecurityGroup{sg1, sg2})
 
 	sgMap, err := api.GetSecurityGroups(context.TODO())
-	c.Assert(err, check.IsNil)
-	c.Assert(sgMap, checker.DeepEquals, types.SecurityGroupMap{"sg1": sg1, "sg2": sg2})
+	require.NoError(t, err)
+	require.Equal(t, types.SecurityGroupMap{"sg1": sg1, "sg2": sg2}, sgMap)
```

Comment:
Does a 'normal' equal work on a complex type? I think it's a `map[string]*SecurityGroup` where `SecurityGroup` is a deep type. Does this need to be `require.EqualValues`?